pkgs/servers/bird/bird-lg.nix

```
@@ -0,0 +1,49 @@
+{ buildGoModule, fetchFromGitHub, lib, symlinkJoin }:
+let
+  generic = { mod, vendorSha, runTests ? true }:
+    buildGoModule rec {
+      name = "bird-lg-${mod}";
+      version = "2022-05-08";
+
+      doCheck = runTests;
+      doDist = false;
+
+      ldflags = [
+        "-s"
+        "-w"
+      ];
+
+      modRoot = mod;
+
+      src = fetchFromGitHub {
+        owner = "xddxdd";
+        repo = "bird-lg-go";
+        rev = "348295b9aa954a92df2cf6b1179846a9486dafc0";
+        sha256 = "sha256-2t8ZP9Uc0sJlqWiJMq3MVoARfMKsuTXJkuOid0oWgyY=";
+      };
+
+      vendorSha256 = vendorSha;
+
+      meta = with lib; {
+        description = "Bird Looking Glass";
+        homepage = "https://github.com/xddxdd/bird-lg-go";
+        license = licenses.gpl3Plus;
+        maintainers = with maintainers; [ tchekda ];
+        platforms = platforms.all;
+      };
+    };
+
+  bird-lg-frontend = generic {
+    mod = "frontend";
+    vendorSha = "sha256-WKuVGiSV5LZrJ8/672TRN6tZNQxdCktHV6nx0ZxCP4A=";
+    # No access to network during checkPhase => whois test fails
+    # https://github.com/xddxdd/bird-lg-go/issues/46
+    runTests = false;
```

Can we disable only the whois tests?

```suggestion
    # No access to network during checkPhase => whois test fails
    # https://github.com/xddxdd/bird-lg-go/issues/46
    doCheck = false;
```